FireRender.Maya.Src/FireRenderMaterialSwatchRender.cpp

```diff
@@ -98,14 +101,19 @@ bool FireRenderMaterialSwatchRender::setupFRNode()
 
 	auto disableSwatchPlug = nodeFn.findPlug("disableSwatch");
 
-	if (disableSwatchPlug.isNull() || !disableSwatchPlug.asBool())
+	bool enableSwatches = false;
+	int iterations = FireRenderGlobalsData::getThumbnailIterCount(&enableSwatches);
```

Comment:
int iterations is declared but never used. Is this supposed to be like that?

Reply:
Fixed